src/freertos_drivers/net_cc32xx/CC32xxWiFi.cxx

```diff
@@ -551,46 +578,21 @@ void CC32xxWiFi::wlan_setup_ap(const char *ssid, const char *security_key,
     HASSERT(strlen(ssid) <= 32);
     HASSERT(strlen(security_key) <= 64);
 
-    int sec_type = security_type_to_simplelink(security_type);
-    uint16_t len;
-    uint16_t config_opt;
+    uint8_t sec_type = security_type_to_simplelink(security_type);
 
-    char old_ssid[32];
-    len = 32;
-    config_opt = WLAN_AP_OPT_SSID;
-    sl_WlanGet(SL_WLAN_CFG_AP_ID, &config_opt , &len, (uint8_t*)old_ssid);
-    // simplify strcmp by null terminating the old_ssid
-    if (len != strlen(ssid) || strncmp(old_ssid, ssid, len))
```

Comment:
how come you are removing these tests? I thought we need to protect the NWP flash from being written too often.

Reply:
The reason is simple.  These values cannot be read from the NWP.  This is prevented for security reasons.  I'll put some commentary in the code.

Reply:
is the non-read true only for the password or also the rest of the settings?